### PR TITLE
vpd preservation for manually flashing page

### LINKED ISSUE #131: Manually Flashing Page is missing VPD preservation
title

https://docs.chrultrabook.com/docs/firmware/manually-flashing.html

### PR DESCRIPTION
**Pull request by ninelore**
Fixes #131

### CODE REVIEW

**Comment by ninelore** [comment]
TODO: Ch341a and Flipper Zero pages are missing this aswell

**Review by ethanaobrien — APPROVED** [review]

**Comment by WeirdTreeThing** [comment]
Should include hwid preservation too

File: src/docs/firmware/manually-flashing.md [hunk 1]
```diff
@@ -4,6 +4,28 @@
...
+
+::: tip
+You don't need to do this if flashing a stock firmware backup created by the Firmware Utility Script; that image already contains the VPD.
+:::
+
+1. For both the options below, we'll need to use the cbfstool (coreboot filesystem) binary, so let's download/extract that:
+    * `wget https://mrchromebox.tech/files/util/cbfstool.tar.gz && tar -zxf cbfstool.tar.gz`
+    * Option 1: Extract VPD from the firmware on device
+        * `sudo flashrom -p ch341a_spi -r badflash.rom`
```
Comment (ethanaobrien) [inline src/docs/firmware/manually-flashing.md:18]: missed this the first time, should be internal

**Review by ethanaobrien — CHANGES_REQUESTED** [review]

File: src/docs/firmware/manually-flashing.md (hunk 1 above)
Comment (ninelore) [reply src/docs/firmware/manually-flashing.md]: fixed

**Review by ethanaobrien — APPROVED** [review]

**Review by WeirdTreeThing — CHANGES_REQUESTED** [review]
as I said before, this should also mention persisting hwid

**Comment by ninelore** [comment]
> as I said before, this should also mention persisting hwid

isnt that included in the (term) **v**ital **p**roduct **d**ata?

**Comment by WeirdTreeThing** [comment]
No. HWID is part of the GBB, not RO_VPD.

**Comment by ethanaobrien** [comment]
https://github.com/MrChromebox/scripts/blob/cde177b9c534cb95d7cff5ffaf0ba5efed50e93d/firmware.sh#L313

https://github.com/MrChromebox/scripts/blob/cde177b9c534cb95d7cff5ffaf0ba5efed50e93d/firmware.sh#L274

**Comment by meghan06** [comment]
@ninelore VPD Preservation is not possible with FZ (Flipper Zero) from my testing. HWID preservation is also untested. 

https://docs.chrultrabook.com/docs/unbricking/unbrick-flipper.html#preparations

**Comment by ninelore** [comment]
@WeirdTreeThing I consider HWID preservation out of scope of this PR since is has not been covered at all by the docs yet.

**Comment by WeirdTreeThing** [comment]
It's two commands...

**Comment by ninelore** [comment]
will redo this from scratch

**Pull request closed without merging**